src/ScrumMaster/Channel/ChannelResult.php

```diff
@@ -95,4 +95,12 @@ private function peopleAssigned(): array
 
         return $people;
     }
+
+    private function replaceKey(array $tickets, string $from = '', string $to = 'None'): array
```

Comment:
I wouldn't use a default value for `$from` neither for `$to`. I rather prefer to force these values explicit where this function is used.
```php
return $this->replaceKey($tickets, $from = '', $to = 'None');
```